src/breeds.ts

```diff
@@ -1304,3 +1305,29 @@ export const dangerousBreeds = [
 ] as const;
 
 export type DangerousBreeds = typeof dangerousBreeds[number];
+
+export const dangerousBreedsSpain: Breeds[] = [
+  'Tosa Inu',
+  'Akita Inu',
+  'Rottweiler',
+  'Fila Brasileiro',
+  'Dogo Argentino',
+  'Pit Bull Terrier',
+  'Staffordshire Bull Terrier',
+  'American Staffordshire Terrier',
+  'Doberman',
+  'American Bully',
+  'American Akita',
+  'American Pit Bull Terrier',
+  'Bordeaux Mastiff',
+  'Tibetan mastiff',
+  'Neapolitan Mastiff',
+  'Perro de Presa Canario',
+  'Perro de Presa Mallorquin',
+  'Bull terrier',
+  'boxer',
```

Comment:
Should the initial of `boxer` be in upper case too?

Reply:
So interestingly in this case, the `boxer` was really lowercased in the breeds list.
We could try to change both occurrences.

Reply:
Done, thanks.